tally/statgram.go

```diff
@@ -60,14 +62,23 @@ func (parser *StatgramParser) ParseStatgram(datagram []byte) Statgram {
 			prefixLen, err := strconv.ParseInt(string(line[1:3]), 16, 0)
```

Comment:
IIRC, this can also be abbreviated as:
```go
if prefixLen, err := strconv.ParseInt(string(line[1:3]), 16, 0); err != nill && int(prefixLen) <= previousLen {
    ...
}
```
When appropriate, according to context and/or whitespace.